circe/src/main/scala/org/http4s/circe/CirceEntityEncDec.scala

```diff
@@ -0,0 +1,3 @@
+package org.http4s.circe
+
+object CirceEntityEncDec extends CirceEntityDecoder with CirceEntityEncoder
```

Comment:
I think `Codec` is a more common term than `EncDec`.  How about `CirceEntityCodec`?

Reply:
Yeah, sounds better.